crypto/dalek/Cargo.toml

```diff
@@ -24,8 +25,8 @@ x25519-dalek = { version = "2.0.0-pre.2", default-features = false, features = [
 # if `mc-crypto-dalek/default` is selected we'll pull in the avx or u64 backends based on the target,
 # otherwise backend selection is up to the user
 
-[target.'cfg(any(target_feature = "avx2", target_feature = "avx"))'.dependencies]
+[target.'cfg(all(any(target_feature = "avx2", target_feature = "avx"), target_arch = "x86_64"))'.dependencies]
```

Comment:
are you sure this is correct? I think not all x86_64 necessarily have avx2

Reply:
yep unless i've flubbed this is `(avx2 || avx) && x86_64` because `avx` extensions are `x86_64` specific and some broken target detection somewhere was trying to invoke the avx2 feature when building for the ledger

Reply:
oh i misread it, i thought this was `avx2 || avx || x86_64`

okay i get what you are doing

sounds good to me, thanks